--- out/test2.out	2024-02-07 15:49:09.337317984 -0500
+++ tests/test2.res	2024-01-09 17:01:27.000000000 -0500
@@ -0,0 +1,6 @@
+* initial money=10 fun=20
+* Monday 1:00 -> 9:00: early work (money=18 fun=20)
+* Monday 9:00 -> 15:00: home alcoholism (money=14 fun=34)
+* Monday 15:00 -> 1:00: night work (money=32 fun=26)
+* Tuesday 1:00 -> 9:00: early work (money=40 fun=26)
+* Tuesday 9:00 -> 15:00: home alcoholism (money=36 fun=40)
